src/shims/unix/fs.rs

```diff
@@ -911,13 +907,16 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
         this.try_unwrap_io_result(result)
     }
 
-    fn macos_stat(
+    fn macos_fbsd_stat(
```

Comment:
All of these will need careful auditing since macos is 64bit-only. It's unclear whether the existing code makes sense for other OSes.